pkg/kubelet/token/token_manager.go

```diff
@@ -175,11 +174,15 @@ func (m *Manager) requiresRefresh(tr *authenticationv1.TokenRequest) bool {
 		klog.ErrorS(nil, "Expiration seconds was nil for token request", "tokenRequest", cpy)
 		return false
 	}
+
 	now := m.clock.Now()
 	exp := tr.Status.ExpirationTimestamp.Time
 	iat := exp.Add(-1 * time.Duration(*tr.Spec.ExpirationSeconds) * time.Second)
 
-	jitter := time.Duration(rand.Float64()*maxJitter.Seconds()) * time.Second
+	// Dynamically set maxJitter to 15% of the token's TTL
```

Comment:
An unconditional 15% seems too high, it could discard hours or days of a longer-lived token's lifetime.

The minimum token lifetime is 10 minutes, if we want to make this dynamic, I'd probably do something like 10% of the token's lifetime or 5 minutes, whichever is less.

Reply:
I'm currently serving in the obligation military and can only use electronic devices in a limited capacity, so I will submit the commit after a month..